README.md

```diff
@@ -6,22 +6,22 @@ This is the officially supported .NET library for using Adyen's APIs.
+| [Transfers API](https://docs.adyen.com/api-explorer/transfers/3/overview)                                  | The Transfers API provides endpoints that you can use to get information about all your transactions, move funds within your balance platform or send funds from your balance platform to a transfer instrument. | [Transfers](Adyen/Service/Transfers)                      | **v3**
+| [Configuration API](https://docs.adyen.com/api-explorer/balanceplatform/2/overview)                        | The Configuration API enables you to create a platform where you can onboard your users as account holders and create balance accounts, cards, and business accounts. | [BalancePlatform](Adyen/Service/BalancePlatform) | **v2** 
+| [Balance Control API](https://docs.adyen.com/api-explorer/BalanceControl/1/overview)                       | The Balance Control API lets you transfer funds between merchant accounts that belong to the same legal entity and are under the same company account.                          | [BalanceControl](Adyen/Service/BalanceControlService.cs)  | **v1** |
+| [Data Protection API](https://docs.adyen.com/development-resources/data-protection-api)                    | Our Data Protection API allows you to process Subject Erasure Requests as mandated in General Data Protection Regulation (GDPR).                                                | [DataProtection](Adyen/Service/DataProtectionService.cs)  | **v1** |
+| [Terminal API (Cloud communications)](https://docs.adyen.com/point-of-sale/choose-your-architecture/cloud) | Our point-of-sale integration.                                                                                                                                                  | [Cloud-based Terminal API](Adyen/Service/TerminalCloudApi.cs)     | Cloud-based Terminal API |      |
+| [Terminal API (Local communications)](https://docs.adyen.com/point-of-sale/choose-your-architecture/local) | Our point-of-sale integration.                                                                                                                                                  | [Local-based Terminal API](Adyen/Service/TerminalLocalApi.cs) | Local-based Terminal API |      |
+| [IPP Terminal Management API](https://docs.adyen.com/api-explorer/postfmapi/1/overview)                    | This API provides endpoints for managing your point-of-sale (POS) payment terminals. You can use the API to obtain information about a specific terminal, retrieve overviews of your terminals and stores, and assign terminals to a merchant account or store. | [POSTerminalManagement](Adyen/Service/POSTerminalManagementService.cs) | **v1**|
```

Comment:
Name changed to IPP terminal management, but names in other columns are still  named POSTerminalManagement. that seems conflicting

Reply:
In the docs indeed is mentioned as [POS Terminal Management API](https://docs.adyen.com/api-explorer/postfmapi/1/overview) so I revert the name